etor/src/main/resources/transformation_definitions.json

```diff
@@ -216,6 +216,25 @@
           "args": {}
         }
       ]
+    },
+    {
+      "name": "ucsdOruRemoveAccessionNumberObservation",
+      "description": "Remove Observations when their OBX-3.4 value is '99717-5' and OBX-3.6 is 'L'",
```

Comment:
I think it'd be helpful to call out this is for ORUs for UCSD

Reply:
done